src/publishProject/projectUtils.ts

```diff
@@ -0,0 +1,172 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as mssql from "vscode-mssql";
+import * as constants from "../constants/constants";
+import { SqlProjectsService } from "../services/sqlProjectsService";
+
+// Shape returned by sqlProjectsService.getProjectProperties (partial, only fields we use)
+export interface ProjectProperties {
+    projectGuid?: string;
+    configuration?: string;
+    outputPath: string;
+    databaseSource?: string;
+    defaultCollation: string;
+    databaseSchemaProvider: string;
+    projectStyle: unknown;
+    targetVersion?: string;
+}
+
+/**
+ * Target platforms for a sql project
+ */
+export const enum SqlTargetPlatform {
```

Comment:
Can you add the same note here about them matching the values in Microsoft.Data.Tools.Schema.SchemaModel.SqlPlatformNames?

Reply:
Updating the note accordingly, these names are how we display the available target platform names in sqlproj UI, not coming from dacfx.